Wikipedia/UI-V5/WMFHomeSectionSchema.m

```diff
@@ -122,7 +128,7 @@ - (void)update:(BOOL)force {
     //Check Tweak
     if (!FBTweakValue(@"Home", @"General", @"Always update on launch", NO)) {
         //Check force flag and minimum relaod time
-        if (!force && [[NSDate date] timeIntervalSinceDate:self.lastUpdatedAt] < WMFHomeMinimumAutomaticReloadTime) {
+        if (!force && self.lastUpdatedAt && [[NSDate date] timeIntervalSinceDate:self.lastUpdatedAt] < WMFHomeMinimumAutomaticReloadTime) {
```

Comment:
this might be out of scope for this PR. we can have a separate discussion w/ Josh since scroll position restoration is tied up in this as well